src/fetch/fetch_api.rs

```diff
@@ -233,9 +232,8 @@ impl L8RequestObject {
             .http_client
             .post(format!("{}/proxy", network_state.forward_proxy_url))
             .header("content-type", "application/json")
-            .header(
-                "ntor-session-id",
-                network_state.init_tunnel_result.ntor_session_id.clone(),
+            .header("int_rp_jwt", network_state.init_tunnel_result.token1.clone())
```

Comment:
Yeah.... token1 & token2. Probably my only criticism on this PR. More description would be good.